pkgs/development/tools/misc/cquery/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub, makeWrapper
+, cmake, llvmPackages, ncurses }:
+
+let
+  src = fetchFromGitHub {
+    owner = "cquery-project";
+    repo = "cquery";
+    rev = "e45a9ebbb6d8bfaf8bf1a3135b6faa910afea37e";
```

Comment:
How about a released version? https://github.com/cquery-project/cquery/releases

Reply:
The latest release version has a few bugs that have since been fixed. This revision works better (at least in the code bases I tested it with).

Reply:
ok. releases as it seems progress fast here anyway.